src/njoy21/input/POWR/Lib3/Card9/Indfis.hpp

```diff
@@ -0,0 +1,8 @@
+struct Indfis{
```

Comment:
Where is the test for `Indfis`?

Reply:
It has no verify method as I have no idea what allowable values are, so there is no test to be performed without further info.